Tree/Binary_tree.py

```
@@ -0,0 +1,29 @@
+class BinaryTree:
+    def __init__(self, data) -> None:
+        self.data = data
+        self.leftChild = None
+        self.rightChild = None
+
+    def insert_leftChild(self, data):
+        if self.leftChild is None:
+            self.leftChild = BinaryTree(data)
+        else:
+            print("Left child already exis.")
```

Raise error instead of print.